lib/stripe_model_callbacks/models/stripe_subscription_schedule.rb

```diff
@@ -35,6 +38,17 @@ def assign_from_stripe(object)
     assign_subscription_schedule_phases(object)
   end
 
+  def cancel_on_stripe
+    to_stripe.cancel
+    update!(canceled_at: Time.zone.now) if respond_to?(:canceled_at)
+    reload_from_stripe!
+    true
+  end
+
+  def cancel_on_stripe!
+    raise ActiveRecord::RecordInvalid, self unless cancel_on_stripe
```

Comment:
Can `cancel_on_stripe` return something else than true og raise an error? :-)

Reply:
Used same pattern as in https://github.com/kaspernj/stripe_model_callbacks/blob/master/app/models/stripe_model_callbacks/application_record.rb

https://github.com/kaspernj/stripe_model_callbacks/blob/master/app/models/stripe_model_callbacks/application_record.rb#L46-L48

https://github.com/kaspernj/stripe_model_callbacks/blob/master/app/models/stripe_model_callbacks/application_record.rb#L57-L59